recipes-extended/xen/files/xc-xt-v4v.patch

```diff
@@ -6,9 +6,9 @@ index ba57e5a..f2073e9 100644
      HYPERCALL(hvm_op),
      HYPERCALL(sysctl),
      HYPERCALL(domctl),
--    HYPERCALL(tmem_op)
-+    HYPERCALL(tmem_op),
-+    HYPERCALL(v4v_op)
+     HYPERCALL(tmem_op),
++    HYPERCALL(v4v_op),
+     [ __HYPERVISOR_arch_1 ] = (hvm_hypercall_t *)paging_domctl_continuation
```

Comment:
Upstream change to the hypercall table has an anchor now, so in order to have the v4v patch apply, it was modified to contain the anchor.